docs/topic-guides/system-concepts/resource-pool.txt

```diff
@@ -0,0 +1,28 @@
+.. _resource-pool:
+
+###############
+ Resource Pool
+###############
+
+Large-scale deep learning requires using different resources, including
+computation resources, memory, network, and storage. Typically, we start
+off prototyping our models with immediately available instance types
+with few accelerators. Once our models are mature enough, we want to
+scale up the training speed with stronger instance types with full
+accelerators and high-performing processors and more instances. In the
+meantime, in order to avoid the network becomes a bottleneck we want to
+let the computation happen as close to data as possible. We also want to
```

Comment:
I don't fully understand this line. Latency impacts throughput, but I didn't think it was that huge. I would probably remove the line about data locality

Reply:
removed